qemu/tests/virtio_fs_share_data.py

```diff
@@ -741,6 +741,14 @@ def check_security_label(file, folder, xattr_name):
                     time.sleep(1)
                     check_security_label(file_share_in_guest, fs_dest, selinux_xattr_name)
 
+                    error_context.context("The list of xattr for the file is empty "
+                                          "in guest, let's check it.", test.log.info)
+                    getfattr_list_cmd = params.get("getfattr_list_cmd")
+                    s, o = session.cmd_status_output(getfattr_list_cmd % file_new_in_guest)
+                    if s:
+                        test.fail("Getting the emply list of xattr failed"
```

Comment:
LGTM, but there seems to be a typo here. emply or empty?

Reply:
Update, thanks.